operator/konflux.bundle.Dockerfile

```diff
@@ -44,6 +44,7 @@ ARG RELATED_IMAGE_SCANNER_V4_DB
 ENV RELATED_IMAGE_SCANNER_V4_DB=$RELATED_IMAGE_SCANNER_V4_DB
 RUN echo "Checking required RELATED_IMAGE_SCANNER_V4_DB"; [[ "${RELATED_IMAGE_SCANNER_V4_DB}" != "" ]]
 
+ARG RELATED_IMAGE_COLLECTOR
```

Comment:
I'm a little confused about this change, and I'm not entirely sure how/if this worked before

Reply:
It was already changed in your last PR: https://github.com/stackrox/stackrox/pull/13350/files#diff-a0bd03066e74bedcc5176486cfb75755731e939937c28281f6dc3cbc7a470953L47-L53.  
That PR never ran Konflux builds (because it didn't have 'konflux' in the branch name or `konflux-build` label set). 

In the merge commit, the Konflux build for the Operator Bundle failed as expected: https://github.com/stackrox/stackrox/runs/36778765375. 

ie it has worked before #13350, then broke, this PR should fix it again.

Reply:
And it does in fact resolve it: https://github.com/stackrox/stackrox/pull/14169/checks?check_run_id=36860868383.

Reply:
Ahh damn, of course, sorry about that.